src/components/TodoAppHeader/TodoAppHeader.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect } from 'react';
+import { ErrorText } from '../../types/ErrorText';
+import { Todo } from '../../types/Todo';
+import cn from 'classnames';
+
+type Props = {
+  todos: Todo[];
+  onAddNewTodo: (_: string) => void;
+  onError: (_: ErrorText) => void;
+  isPosting: boolean;
+  title: string;
+  setTitle: (_: string) => void;
+  inputRef: React.RefObject<HTMLInputElement>;
+  onTodoCheck: (_: Todo) => void;
```

Comment:
Why do you call arguments `_`? You should use it only if you don't need the first one and want to use others, but it's not the case here